packages/block-editor/src/components/list-view/style.scss

```diff
@@ -475,22 +475,16 @@ $block-navigation-max-indent: 8;
 		($block-navigation-max-indent - 1);
```

Comment:
I think this one will also need updating? Currently the max indent level appears to be off:

<img width="870" alt="image" src="https://github.com/WordPress/gutenberg/assets/14988353/11350eb3-5b25-42c5-bb5b-086eb1affafa">

Reply:
For sure. We might consider increasing `$block-navigation-max-indent` as a part of this work too.